shared/src/mobileMain/kotlin/dev/johnoreilly/confetti/SessionsViewModel.kt

```diff
@@ -93,7 +92,7 @@ open class SessionsViewModel : KMMViewModel(), KoinComponent {
         val timeZone = sessionsData.config.timezone.toTimeZone()
         val sessionsMap =
             sessionsData.sessions.nodes.map { it.sessionDetails }.groupBy { it.startsAt.date }
-        val speakers = sessionsData.speakers.map { it.speakerDetails }
+        val speakers = sessionsData.speakers.nodes.map { it.speakerDetails }
```

Comment:
That's an issue, it's going to break apps currently in production. We're a real product now, we need to go through deprecation lifecycle.

Reply:
Good point :) Just pushed a new field `speakersPage` and marked `speakers` as deprecated.